File: array/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i = 0
        j = len(numbers)-1
        ret = []
        
        while i<j:
            sm = numbers[i]+numbers[j]
            if sm == target:
                ret.append(i+1)
                ret.append(j+1)
                return ret
            elif sm >target:
                j-=1
            else:
                i+=1
        return []
        

File: binary-search/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int left = 0;
        int right = nums.length - 1; 
        while (left < right){
            int sum = left + right; 
            if (sum > target){
                right--;
            }else if(sum < target){
                left++; 
            }else{
                return [left,right]; 
            }
        }
    }
}